utils/afl_untracer/afl-untracer.c

```diff
@@ -156,7 +156,7 @@ void read_library_information(void) {
         *e = 0;
         if (n[strlen(n) - 1] == '\n') n[strlen(n) - 1] = 0;
 
-        liblist[liblist_cnt].name = strdup(n);
+        liblist[liblist_cnt].name = strdup((char *)n);
```

Comment:
Should the result of strdup be casted to `u8*` then?

Reply:
yes it s fair.